aspnetcore/fundamentals/configuration/index.md

```diff
@@ -5,1850 +5,16 @@ description: Learn how to use the Configuration API to configure AppSettings in
-
-<a name="dhc"></a>
-
-## Default host configuration
-
-For details on the default configuration when using the [Web Host](xref:fundamentals/host/web-host), see the [ASP.NET Core 2.2 version of this topic](?view=aspnetcore-2.2&preserve-view=true).
-
-* Host configuration is provided from:
-  * Environment variables prefixed with `DOTNET_` (for example, `DOTNET_ENVIRONMENT`) using the [Environment Variables configuration provider](#environment-variables). The prefix (`DOTNET_`) is stripped when the configuration key-value pairs are loaded.
-  * Command-line arguments using the [Command-line configuration provider](#clcp).
-* Web Host default configuration is established (`ConfigureWebHostDefaults`):
-  * Kestrel is used as the web server and configured using the app's configuration providers.
-  * Add Host Filtering Middleware.
-  * Add Forwarded Headers Middleware if the `ASPNETCORE_FORWARDEDHEADERS_ENABLED` environment variable is set to `true`.
-  * Enable IIS integration.
-
-## Other configuration
-
-This topic only pertains to *app configuration*. Other aspects of running and hosting ASP.NET Core apps are configured using configuration files not covered in this topic:
-
-* `launch.json`/`launchSettings.json` are tooling configuration files for the Development environment, described:
-  * In <xref:fundamentals/environments#development>.
-  * Across the documentation set where the files are used to configure ASP.NET Core apps for Development scenarios.
-* `web.config` is a server configuration file, described in the following topics:
-  * <xref:host-and-deploy/iis/index>
-  * <xref:host-and-deploy/aspnet-core-module>
-
-Environment variables set in `launchSettings.json` override those set in the system environment.
-
-For more information on migrating app configuration from earlier versions of ASP.NET, see <xref:migration/proper-to-2x/index#store-configurations>.
-
-## Add configuration from an external assembly
-
-An <xref:Microsoft.AspNetCore.Hosting.IHostingStartup> implementation allows adding enhancements to an app at startup from an external assembly outside of the app's `Startup` class. For more information, see <xref:fundamentals/configuration/platform-specific-configuration>.
-
-## Additional resources
-
-* [Configuration source code](https://github.com/dotnet/runtime/tree/main/src/libraries/Microsoft.Extensions.Configuration)
-* [WebApplicationBuilder source code](https://github.com/dotnet/aspnetcore/blob/v6.0.1/src/DefaultBuilder/src/WebApplicationBuilder.cs)
-* [View or download sample code](https://github.com/dotnet/AspNetCore.Docs/tree/main/aspnetcore/fundamentals/configuration/index/samples) ([how to download](xref:index#how-to-download-a-sample))
-* <xref:fundamentals/configuration/options>
-* <xref:blazor/fundamentals/configuration>
-
-:::moniker-end
+[!INCLUDE[](~/includes/configuration/index6.md)]
```

Comment:
@tdykstra @wadepickett  for now on, all updates should be IFBV (Intermediate or Include File By Version). That will make it trivial to:

* start the next version
* Promote the released version at GA

cc @guardrex